src/android/HockeyApp.java

```diff
@@ -27,14 +31,70 @@
 import java.lang.Thread;
 import java.text.SimpleDateFormat;
 import java.util.Calendar;
+import java.util.ArrayList;
+
+import android.app.Activity;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.view.View;
 
 public class HockeyApp extends CordovaPlugin {
 
+    public static final long XWALK_SCREENSHOT_WAIT_MS = 5000;
+    public static final String XWALK_SCREENSHOT_CAPTURE_MSG = "captureXWalkBitmap";
+    public static final String XWALK_SCREENSHOT_BITMAP_MSG = "onGotXWalkBitmap";
+    
     public static boolean initialized = false;
     public static String appId;
+    public static Object monitor = new Object();
+    public static volatile Bitmap bitmap;
     
     private ConfiguredCrashManagerListener crashListener;
+    
+    // Integration with Crosswalk requires:
+    // - Crosswalk engine version 18 or later
+    // - Latest cordova-plugin-crosswalk-webview plugin     
+    private Bitmap getBitmap() {
+        bitmap = null;
+        boolean isCrosswalk = false;
+        try {
+            Class.forName("org.crosswalk.engine.XWalkWebViewEngine");
```

Comment:
Unfortunately it seems this change does not work. The class name of `webView` is `CordovaWebViewImpl`. It seems it is the internal variable `webView.engine` which is of type `XWalkWebViewEngine` but this code would not be able to test for that. I'll leave this one as it was and fix the other issue.